src/renderer/screens/exchange/index.js

```diff
@@ -103,7 +103,7 @@ const SelectProvider = () => {
 
   return (
     <>
-      <TrackPage category="Exchange" name="SelectProvider" />
+      <TrackPage category="Buy" name="SelectProvider" />
```

Comment:
see internal discussion. possible issue on this one but i don't have the big picture to know if this was requested.

Reply:
Internally ack. Good to go.